ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Impl/Validation.hs

```diff
@@ -106,6 +106,11 @@ validateAndReopen validateEnv registry valPol = wrapFsError (Proxy @blk) $ do
     ValidateEnv { hasFS, tracer, cacheConfig, chunkInfo } = validateEnv
     cacheTracer = contramap TraceCacheEvent tracer
 
+decorateValidateTracer :: ChunkNo
```

Comment:
What is this `ChunkNo` intended to represent? Is this function used somewhere else?

Reply:
TODO: add a comment and move the definition inside `validate`.

Reply:
Done